backend.go

```diff
@@ -7,29 +7,42 @@ import (
 	"time"
 )
 
+type BackendType interface {
+	any | []byte
+}
+
```

Comment:
Seems like an unused constraint. Did you want to use it in `OpenStore[T any]` or should we remove?

Reply:
Yeah, I think I forgot to change it. Resolved.